src/lib/cometvisu-client-openhab.js

```diff
@@ -0,0 +1,112 @@
+/* cometvisu.js (c) 2010 by Christian Mayer [CometVisu at ChristianMayer dot de]
```

Comment:
Can you explain a bit about the new protocoll and why it needs a new communication lib / can't be using a unified communication lib?

And as it's "your" file you can place yourself in the first line, of course :)